com.examples.exercise/src/main/java/com/examples/exercise/DbMongo.java

```diff
@@ -33,10 +34,21 @@ public void updateDB(String id, String name) {
 	}
 
 	public boolean exists(String id) {
+		BasicDBObject query = new BasicDBObject();
```

Comment:
One of the requirements of the issue was that exists must be implemented in terms of takeStudentById, not the other way round: it does not make sense to first check whether a Student exists and then retrieve it with another query